tests/unit/mgclient_value.cpp

```diff
@@ -230,11 +230,11 @@ TEST(ValueTest, MapIterate) {
   map.Insert("key 3", Value(3.0));
 
   std::vector<std::pair<std::string, ConstValue>> values;
-  for (const auto &[key, value] : map) {
+  for (const auto [key, value] : map) {
```

Comment:
Why did you remove reference?
You can move now key and value in the emplace_back call bellow but don't see any other reason why you would do this now?

Reply:
AppleClang 12 says
```
[ 62%] Building CXX object tests/CMakeFiles/unit_mgclient_value.dir/unit/mgclient_value.cpp.o
/Users/runner/work/mgclient/mgclient/tests/unit/mgclient_value.cpp:121:20: warning: loop variable 'value' is always a copy because the range of type 'mg::List' does not return a reference [-Wrange-loop-analysis]
  for (const auto &value : list) {
                   ^
/Users/runner/work/mgclient/mgclient/tests/unit/mgclient_value.cpp:121:8: note: use non-reference type 'mg::ConstValue'
  for (const auto &value : list) {
       ^~~~~~~~~~~~~~~~~~~
```

Reply:
That's a nice warning.

Reply:
Yep :smile: